tank_lab_to_nwb/convert_towers_task/convert_towers.py

```diff
@@ -8,39 +8,37 @@
 
 n_jobs = 1  # number of parallel streams to run
 
-base_path = "D:/Neuropixels/TowersTask"
+base_path = "D:/Neuropixels"
 # base_path = "/mnt/scrap/cbaker239/Neuropixels/TowersTask"
 
 # Manual list of selected sessions that cause problems with the general functionality
 exlude_sessions = []
 
-session_strings = ["PoissonBlocksReboot_cohort1_VRTrain6_E75_T_20181105"]
+virmen_session_strings = ["TowersTask/PoissonBlocksReboot_cohort1_VRTrain6_E75_T_20181105"]
+spikeglx_session_strings = ["Neuropixels/A256_bank1_2020_09_30/"
+                            "A256_bank1_2020_09_30_g0/A256_bank1_2020_09_30_g0_t0"]
 nwbfile_paths = []
-for j, session in enumerate(session_strings):
-    session_strings[j] = os.path.join(base_path, session_strings[j])
-    nwbfile_paths.append(os.path.join(base_path, session) + "_local_stub.nwb")
+for j, (virmen_session, spikeglx_session) in enumerate(zip(virmen_session_strings, spikeglx_session_strings)):
+    virmen_session_strings[j] = os.path.join(base_path, virmen_session_strings[j])
+    spikeglx_session_strings[j] = os.path.join(base_path, spikeglx_session_strings[j])
+    nwbfile_paths.append(os.path.join(base_path, virmen_session) + "_local_stub.nwb")  # name defaults to virmen session
 
 
-def run_tower_conv(session, nwbfile_path):
+def run_tower_conv(virmen_session, spikeglx_session, nwbfile_path):
     """Conversion function to be run in parallel."""
     if os.path.exists(base_path):
-        print(f"Processsing {session}...")
+        print(f"Processsing {virmen_session}...")
         if not os.path.isfile(nwbfile_path):
-            # construct input_args dict according to input schema
             input_args = dict(
-                SpikeGLXRecording=dict(
-                    file_path="D:/Neuropixels/Neuropixels/A256_bank1_2020_09_30/"
-                    "A256_bank1_2020_09_30_g0/A256_bank1_2020_09_30_g0_t0.imec0.ap.bin"
-                ),
-                TowersPosition=dict(folder_path=session)
+                SpikeGLXRecording=dict(file_path=spikeglx_session+".imec0.ap.bin"),
+                TowersPosition=dict(folder_path=virmen_session)
             )
 
             converter = TowersNWBConverter(**input_args)
             metadata = converter.get_metadata()
 
             # Session specific metadata
             metadata['NWBFile'].update(session_description="")
-            metadata['Subject'].update(species="Mus musculus")
```

Comment:
Why is this being removed?

Reply:
Not sure if they use mice or rats; virmen file doesn't indicate. Was hoping subject database would indicate one way or the other